configs/config-aes-gcm-tls1_3.h

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
I have added a minimal compilation test. I am not quite sure what we want to test here since we are disabling most features - what is your opinion?

IMO if we could somehow have a functionality test using `mini_client` against `openssl s_server` that would be best.